From PeterHuewe@gmx.de  Thu Jan 14 16:59:35 2010
From: Peter Hüwe <PeterHuewe@gmx.de>
Date: Fri, 8 Jan 2010 11:44:38 +0100
Subject: Staging: rtl8192u: Add select WEXT_PRIV to Kconfig to prevent build failure
To: Simon Horman <horms@verge.net.au>
Cc: devel@linuxdriverproject.org, "Greg Kroah-Hartman" <gregkh@suse.de>, Randy Dunlap <randy.dunlap@oracle.com>, linux-kernel@vger.kernel.org, Mauro Carvalho Chehab <mchehab@redhat.com>
Message-ID: <201001081144.39138.PeterHuewe@gmx.de>


From: Peter Huewe <peterhuewe@gmx.de>
Date: Fri, 8 Jan 2010 09:10:13 +0100

Without WEXT_PRIV set the driver fails to build due to unknown fields in
the iw_handler_def struct.
Those fields are enclosed in WEXT_PRIV conditionals in the prototype
of iw_handler_def in include/net/iw_handler.h

Signed-off-by: Peter Huewe <peterhuewe@gmx.de>
Acked-by: Simon Horman <horms@verge.net.au>
Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>

---
 drivers/staging/rtl8192u/Kconfig |    3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

--- a/drivers/staging/rtl8192u/Kconfig
+++ b/drivers/staging/rtl8192u/Kconfig
@@ -1,6 +1,7 @@
 config RTL8192U
 	tristate "RealTek RTL8192U Wireless LAN NIC driver"
 	depends on PCI && WLAN && USB
-	depends on WIRELESS_EXT
+	select WIRELESS_EXT
+	select WEXT_PRIV
 	default N
 	---help---
